src/osdatahub/FeaturesAPI/features_api.py

```diff
@@ -45,12 +42,13 @@ class FeaturesAPI:
         "count": 100,
     }
 
-    def __init__(self, key: str, product_name: str, extent: Extent,
+    def __init__(self, key: str, product_name: str, extent: Extent, new_api: bool = False,
```

Comment:
The code also infers the API type. There is no way to detect what api type they're using until the response is returned. This was just in case people wanted to add it, though to be honest it's somewhat redundant